packages/rightdown/src/core/orchestrator.ts

```
@@ -0,0 +1,202 @@
+import { readFileSync, existsSync } from 'node:fs';
+import { Result, success, failure, makeError, type AppError } from '@outfitter/contracts';
+import { RIGHTDOWN_ERROR_CODES } from './errors.js';
+import { type RightdownConfig } from './types.js';
+import { type IFormatter } from '../formatters/base.js';
+import { AstProcessor, type CodeBlock } from '../processors/ast.js';
```

ESLint

[error] 'CodeBlock' is defined but never used.

(@typescript-eslint/no-unused-vars)